### PR TITLE
Fix build warnings and clippy warnings

### PR DESCRIPTION
**Pull request by dhilipsiva**
## Pull Request
 
Fix Cargo Build Warning and Clippy Warnings

### Checklist

- [x] I have tested the changes locally.
- [x] I have reviewed the code changes.

### CODE REVIEW

File: bin/src/server/media/runtime_worker.rs [hunk 1]
```diff
@@ -58,7 +57,9 @@ impl<ES: 'static + MediaEdgeSecure> WorkerInner<Owner, ExtIn, ExtOut, Channel, E
         let worker = MediaServerWorker::new(
             index,
             cfg.node.node_id,
-            random(),
+            // random(),
+            // NOTE: Not sure if session was supposed to e used here instead of random(); Input Required.
+            cfg.session,
```
[the remark this answers is not in the document]
Comment (giangndm) [reply bin/src/server/media/runtime_worker.rs:62]: @dhilipsiva It is my wrong, and your's changed is correct

File: packages/audio_mixer/src/lib.rs [hunk 2]
```diff
@@ -132,7 +132,7 @@ impl<Src: Debug + Clone + Eq + Hash> AudioMixer<Src> {
         for (i, slot) in self.outputs.iter().enumerate() {
             if let Some(OutputSlotState { audio_level, source }) = slot {
                 if let Some((_, _, lowest_slot_audio_level)) = &mut lowest {
-                    if *audio_level < *lowest_slot_audio_level || (*audio_level == *lowest_slot_audio_level) {
```
Comment (giangndm) [inline packages/audio_mixer/src/lib.rs:135]: Seems to I missed some logic here but it not clearly then can you put TODO here? We need to process some case we have same audio_level. Just check with smaller only

File: packages/transport_webrtc/src/transport.rs [hunk 3]
```diff
@@ -45,7 +45,7 @@ mod whip;
 pub enum VariantParams<ES> {
     Whip(RoomId, PeerId),
     Whep(RoomId, PeerId),
-    Webrtc(String, ConnectRequest, Arc<ES>),
+    Webrtc(String, Box<ConnectRequest>, Arc<ES>),
```
Comment (giangndm) [inline packages/transport_webrtc/src/transport.rs:48]: Why we need to wrapping inside Box? Is it over enum size?

**Review by giangndm — COMMENTED** [review]
Thank you for very carefully merge request, I have some comments and questions, let discuss more in details.

File: packages/transport_webrtc/src/transport.rs (hunk 3 above)
Comment (dhilipsiva) [reply packages/transport_webrtc/src/transport.rs:48]: @giangndm Yes, clippy complained that the Enum member sizes had a wider gap:

```
warning: large size difference between variants
  --> packages/transport_webrtc/src/transport.rs:45:1
   |
45 | / pub enum VariantParams<ES> {
46 | |     Whip(RoomId, PeerId),
   | |     -------------------- the second-largest variant contains at least 48 bytes
47 | |     Whep(RoomId, PeerId),
48 | |     Webrtc(String, ConnectRequest, Arc<ES>),
   | |     --------------------------------------- the largest variant contains at least 264 bytes
49 | | }
   | |_^ the entire enum is at least 264 bytes
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant
   = note: `#[warn(clippy::large_enum_variant)]` on by default
help: consider boxing the large fields to reduce the total size of the enum
```

**Comment by dhilipsiva** [comment]
@giangndm Thank you for your response. I have addressed all your questions. Please let me know if you have more questions or feedbacks.

File: packages/audio_mixer/src/lib.rs (hunk 2 above)
Comment (dhilipsiva) [reply packages/audio_mixer/src/lib.rs:135]: Added a TODO logic.

File: packages/transport_webrtc/src/transport.rs (hunk 3 above)
Comment (dhilipsiva) [reply packages/transport_webrtc/src/transport.rs:48]: @giangndm Should I change it back and silence the clippy instead?

Comment (giangndm) [reply packages/transport_webrtc/src/transport.rs:48]: @dhilipsiva Yes, I think we shouldn't to use Box here for don't confusing other developer. With box some guy will ask why it is needed.

Comment (dhilipsiva) [reply packages/transport_webrtc/src/transport.rs]: Removed the Box now. And silenced clippy.

**Comment by dhilipsiva** [comment]
@giangndm I have updated the code as per your feedbacks. Please let me know if I need to do more changes.

**Review by giangndm — APPROVED** [review]
I will merge it now. Thank you for great contribution.

**Pull request merged**